ts/features/itwallet/issuance/screens/ItwIssuanceCredentialFailureScreen.tsx

```diff
@@ -44,9 +53,29 @@ type ContentViewProps = { failure: CredentialIssuanceFailure };
  */
 const ContentView = ({ failure }: ContentViewProps) => {
   const machineRef = ItwCredentialIssuanceMachineContext.useActorRef();
+  const storedCredential =
+    ItwCredentialIssuanceMachineContext.useSelector(selectCredential);
 
-  const closeIssuance = () => machineRef.send({ type: "close" });
-  const retryIssuance = () => machineRef.send({ type: "retry" });
+  const closeIssuance = () => {
+    machineRef.send({ type: "close" });
+    trackWalletCreationFailed({
+      reason: failure.reason as string,
+      cta_category: "custom_2",
+      cta_id: I18n.t(
+        "features.itWallet.issuance.genericCredentialError.secondaryAction"
+      )
+    });
+  };
+  const retryIssuance = () => {
+    machineRef.send({ type: "retry" });
+    trackWalletCreationFailed({
+      reason: failure.reason as string,
+      cta_category: "custom_1",
+      cta_id: I18n.t(
+        "features.itWallet.issuance.genericCredentialError.primaryAction"
+      )
+    });
+  };
```

Comment:
I'm not sure that we need to use the `reason` property of the failure here. I think we should use the `type` property.
@LazyAfternoons can you confirm this?